styles/styles.css

```diff
@@ -110,6 +110,11 @@ p {
   line-height: 22px;
 }
 
+p.warning,
```

Comment:
I noticed there's a habit of adding CSS into styles.css although it's not used across the site. So I'd say unless it's used in multiple places, don't put that CSS in here. 
On the other hand, we also have duplicated CSS across blocks which would make sense to move into styles.css.

Reply:
.warning is now part of dialog styles